VmAgent.Core/Interfaces/DockerContainerEngine.cs

```diff
@@ -415,22 +415,14 @@
             return sessionHost;
         }
 
-        public static string GetImageNameFromContainerImageDetails(ContainerImageDetails imageDetails)
```

Comment:
>GetImageNameFromContainerImageDetails [](http://example.com/codeflow?start=29&length=37)

Does the image name contain the registry as well?

Reply:
yes it does, ignore

Reply:
The output of this method will contain the registry name as well